srf-weather-connector/processes/srfWeatherForecast.p.json

```diff
@@ -0,0 +1,141 @@
+        "path" : "/geolocationNames",
+        "headers" : {
+          "Accept" : "*/*",
+          "Authorization" : "ivy.var.SrfWeatherConnector_Token"
+        },
+        "clientId" : "abb4be9f-e610-498b-84db-a60dddc472ba",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "queryParams" : {
+          "limit" : "",
+          "name" : "in.location.locationName",
+          "zip" : "in.location.zip"
+        },
+        "statusErrorCode" : "ivy:error:rest:client",
+        "resultType" : "com.fasterxml.jackson.databind.JsonNode",
+        "responseCode" : [
+          "import ch.srgssr.api.srf.meteo.v2.client.GeolocationNamesSearch;",
+          "import com.fasterxml.jackson.databind.node.ObjectNode;",
+          "import com.fasterxml.jackson.databind.node.ObjectNode;",
+          "import com.fasterxml.jackson.databind.JsonNode;",
+          "import com.fasterxml.jackson.databind.node.ArrayNode;",
+          "",
+          "if (!result.isEmpty()) {",
+          "ArrayNode arrayNode = result as ArrayNode;",
+          "  JsonNode firstNode = arrayNode.get(0);",
+          "  ObjectNode objectNode = firstNode as ObjectNode;",
+          "  in.location.locationName = objectNode.get(\"description_short\").asText();",
+          "  ObjectNode geoNode = objectNode.get(\"geolocation\") as ObjectNode;",
+          "  in.location.geolocationId = geoNode.get(\"id\").asText();",
+          "  in.location.locationFound = true;",
+          "}"
+        ]
+      },
+      "visual" : {
+        "at" : { "x" : 208, "y" : 64 }
+      },
+      "connect" : { "id" : "f8", "to" : "f7" }
+    }, {
+      "id" : "f5",
+      "type" : "RestClientCall",
+      "name" : "getForecastPointWeekly",
+      "config" : {
+        "path" : "/forecastpoint/{geolocationId}",
+        "headers" : {
+          "Accept" : "*/*",
+          "Authorization" : "ivy.var.SrfWeatherConnector_Token"
+        },
+        "clientId" : "abb4be9f-e610-498b-84db-a60dddc472ba",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "statusErrorCode" : "ivy:error:rest:client",
+        "templateParams" : {
+          "geolocationId" : "in.location.geolocationId"
+        },
+        "resultType" : "ch.srgssr.api.srf.meteo.v2.client.ForecastPointWeek",
+        "responseCode" : [
+          "in.forcastPointWeek = result;",
+          "in.day = result.days.get(0);"
+        ]
+      },
+      "visual" : {
+        "at" : { "x" : 456, "y" : 64 }
+      },
+      "connect" : { "id" : "f2", "to" : "f1" }
+    }, {
+      "id" : "f7",
+      "type" : "Alternative",
+      "name" : "is geolocationID empty?",
+      "visual" : {
+        "at" : { "x" : 336, "y" : 64 },
+        "labelOffset" : { "x" : 14, "y" : 34 }
+      },
+      "connect" : [
+        { "id" : "f6", "to" : "f5", "condition" : "in.location.geolocationId != \"\"" },
+        { "id" : "f11", "to" : "f10", "via" : [ { "x" : 336, "y" : 144 } ], "condition" : "in.location.geolocationId == \"\"" }

```

Comment:
I think you only have to define the first condition, the second is then the "else" case.